class/defaults.yml

```diff
@@ -1,15 +1,34 @@
 parameters:
   openshift4_logging:
     namespace: openshift-logging
-    version: '5.5'
+    version: '5.6'
```

Comment:
Personally, I would probably separate the upgrade to default to cluster-logging 5.6 and adding support for the Loki stack, since that would make each PR easier to review and would make the changelog better.